content/api/authentication/systemregister/create/_index.en.md

```
@@ -0,0 +1,176 @@
+}
+```
+
+
+## Error Codes
+
+| Error Code     | Status Code | Error Message      | Detailed Description   |
+|----------------|-------------|--------------------|------------------------|
+| AUTH.VLD-00000 | 400 | the org number identifier is not valid ISO6523 identifier | The organization identifier must be 0192, f.ex 0192:991825827 |
+| AUTH.VLD-00001 | 400 | The system id does not match the format orgnumber_xxxx...  | The systemid is expected to be in the format vendororgnumber_xxxxxx |
+| AUTH.VLD-00002 | 400 | The system id already exists | The system id is already taken |
+| AUTH.VLD-00003 | 400 | One or all the resources in rights is not found in altinn's resource register | Any service outside altinn must be registered as a resource in altinn's resource register. Either the service provider has failed to register the resource or is specified wrong in the system register request. 
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)